README.md

```diff
@@ -14,36 +14,68 @@ to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:redis_mutex, "~> 0.4.0"}
+    {:redis_mutex, "~> 1.0"}
   ]
 end
 ```
 
 
 ## Usage
 
-1. Set the `redis_url` in your `config.exs`
+### Adding `RedisMutex` to your application's supervision tree
+
+Add `RedisMutex` to your application's supervision tree.
+
+```elixir
+  @impl Application
+  def start(_type, _args) do
+    children = [other_children() | RedisMutex]
```

Comment:
I know this is just docs, but this would create an improper list with RedisMutex as the tail, and I don't think Supervisor would accept that. Maybe flipping these around would make things a little clearer?
```suggestion
    children = [RedisMutex | other_children()]
```

Reply:
Yes, thank you. That's a good point. I'll update this.

Reply:
Updated